app/src/types/api-models.ts

```diff
@@ -39,22 +39,28 @@ export type SendInntektsmeldingRequestDto = {
   aktorId: string;
   ytelse: Ytelsetype;
   arbeidsgiverIdent: string;
-  telefonnummer: string;
+  kontaktperson: KontaktpersonDto;
   startdato: string;
   inntekt: number;
   refusjonsperioder: RefusjonsperiodeRequestDto[];
   bortfaltNaturaltytelsePerioder: NaturalytelseRequestDto[];
 };
 
-type RefusjonsperiodeRequestDto = {
+type KontaktpersonDto = {
+  telefonnummer: string;
+  navn: string;
+};
+
+export type RefusjonsperiodeRequestDto = {
   fom: string;
-  tom: string;
+  tom?: string;
   beløp: number;
 };
 
-type NaturalytelseRequestDto = {
+export type NaturalytelseRequestDto = {
   fom: string;
-  tom: string;
+  tom?: string;
```

Comment:
skal vi gå for "fraOgMed" og "tilOgMed" istedenfor forkortelsene? `tom` kan liksom bety noe annet, og vi bruker `fraOgMed` et par andre steder.

Reply:
For min del kan vi godt det. Må forandre i BE da